webapp/.eslintrc.json

```diff
@@ -12,6 +12,7 @@
   "globals": {
     "assert": true,
     "fixture": true,
+    "flush": true,
```

Comment:
Looks like we should also add `expect` here and remove `expect` from:
https://github.com/web-platform-tests/wpt.fyi/blob/06b80f2cf093ba41becf318bdd661890d8f82b88/webapp/components/test/test-runs-query.html#L13

Reply:
Done.